### PR TITLE
[TRUS-961] - CYA - Assets Property or Land

### PR DESCRIPTION
**Pull request by jassalrichy**
[TRUS-961] - CYA - Assets Property or Land

### CODE REVIEW

File: app/controllers/property_or_land/PropertyOrLandAnswerController.scala [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                                      view: PropertyOrLandAnswersView,
+                                      countryOptions: CountryOptions,
+                                      val controllerComponents: MessagesControllerComponents
+                                     )(implicit ec: ExecutionContext) extends FrontendBaseController with I18nSupport {
+
+  private def actions(index: Int, draftId: String) =
+    identify andThen
+      getData(draftId) andThen
+      requireData // andThen
+//      requiredAnswer(RequiredAnswer(PropertyOrLandAddressPage(index), routes.PropertyOrLandAddressController.onPageLoad(NormalMode, index, draftId)))
```
Comment (adamconder) [inline app/controllers/property_or_land/PropertyOrLandAnswerController.scala]: @jassalrichy Commented code?

File: app/controllers/property_or_land/PropertyOrLandAnswerController.scala [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+      val sections = Seq(
+        AnswerSection(
+          None,
+          Seq(
+            answers.whatKindOfAsset(index),
+            answers.propertyOrLandAddressYesNo(index),
+            answers.propertyOrLandAddressUkYesNo(index),
+            answers.propertyOrLandUKAddress(index),
+            answers.propertyOrLandInternationalAddress(index),
+            answers.propertyOrLandDescription(index),
```
Comment (adamconder) [inline app/controllers/property_or_land/PropertyOrLandAnswerController.scala]: Do we want description in the same line as addressYesNo so it appears in the order they're asked?

Comment (jassalrichy) [reply app/controllers/property_or_land/PropertyOrLandAnswerController.scala]: Description will still appear in the correct order as long as the clean up logic has been applied correctly.

File: app/controllers/property_or_land/PropertyOrLandAnswerController.scala (hunk 1 above)
Comment (jassalrichy) [reply app/controllers/property_or_land/PropertyOrLandAnswerController.scala]: Corrected to redirect when answer page is hit directly without an answer for "Does the property or land have an address?"

File: app/controllers/property_or_land/PropertyOrLandAnswerController.scala (hunk 2 above)
Comment (adamconder) [reply app/controllers/property_or_land/PropertyOrLandAnswerController.scala]: Ahh yes! You're correct looking back at the site map again.

**Pull request merged**